client/devdocs/design/search-collection.jsx

```diff
@@ -102,10 +106,12 @@ const Collection = ( {
 
 		return (
 			<div>
-				<DocsExampleWrapper name={ exampleName } unique={ !! component } url={ exampleLink }>
-					{ example }
-				</DocsExampleWrapper>
-				{ component && <ReadmeViewer readmeFilePath={ readmeFilePath } /> }
+				<button onClick={ scroll } onKeyPress={ scroll }>
```

Comment:
Buttons shouldn't contain interactive children, so I don't think we should wrap it with a button (and as you note, divs will throw a linting error). In addition to console errors, this would also apply new styles to the children:

Master | Branch
------------ | -------------
<img width="1445" alt="before" src="https://user-images.githubusercontent.com/942359/88665240-565b5580-d0ac-11ea-8828-b42df11fe5cf.png"> | <img width="1443" alt="after" src="https://user-images.githubusercontent.com/942359/88665254-5e1afa00-d0ac-11ea-8a77-63f89bff9691.png">


In testing production, I noticed that _only_ the header of `DocsExampleWrapper` should link to the individual page. As such, we could add a new `onClick` prop to `DocsExampleWrapper`, pass `scroll` to it in `Collection`, and then pass it to `renderTitle()`. Do you want to give that a try?

Reply:
Yeah, for sure!  I'll try this out now.